<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DM3dClass</class>
 <widget class="QMainWindow" name="DM3dClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DM3d</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbrir_OFF"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <widget class="QMenu" name="menuModos">
     <property name="title">
      <string>Modos</string>
     </property>
     <addaction name="actionMallado"/>
     <addaction name="actionVertices"/>
     <addaction name="actionRelleno"/>
     <addaction name="actionBoundingBox"/>
     <addaction name="actionNormales"/>
    </widget>
    <addaction name="menuModos"/>
    <addaction name="separator"/>
    <addaction name="actionActivar_Fog"/>
    <addaction name="actionZ_Buffer"/>
    <addaction name="actionBackface_Culling"/>
    <addaction name="separator"/>
    <addaction name="actionRotar"/>
    <addaction name="actionTransladar"/>
    <addaction name="separator"/>
    <addaction name="actionInformacion_del_Objeto"/>
    <addaction name="separator"/>
    <addaction name="actionOrtogonal"/>
    <addaction name="actionPerspectiva"/>
    <addaction name="separator"/>
    <addaction name="actionNormal_counter_clockwise"/>
    <addaction name="actionNormal_clockwise"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTexturas">
    <property name="title">
     <string>Texturas</string>
    </property>
    <addaction name="actionActivar_Textura"/>
    <addaction name="actionCargar_Textura"/>
    <addaction name="separator"/>
    <addaction name="actionEsferico"/>
    <addaction name="actionCilindrico"/>
    <addaction name="separator"/>
    <addaction name="actionLinear"/>
    <addaction name="actionNeares"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuOpciones"/>
   <addaction name="menuTexturas"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir_OFF"/>
   <addaction name="separator"/>
   <addaction name="actionVertices"/>
   <addaction name="actionMallado"/>
   <addaction name="actionRelleno"/>
   <addaction name="actionBoundingBox"/>
   <addaction name="actionNormales"/>
   <addaction name="separator"/>
   <addaction name="actionActivar_Fog"/>
   <addaction name="actionZ_Buffer"/>
   <addaction name="separator"/>
   <addaction name="actionRotar"/>
   <addaction name="actionTransladar"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAbrir_OFF">
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/open.png</normaloff>:/Imagenes/Iconos/iconos/open.png</iconset>
   </property>
   <property name="text">
    <string>Abrir OFF</string>
   </property>
  </action>
  <action name="actionMallado">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/linepointer.png</normaloff>:/Imagenes/Iconos/iconos/linepointer.png</iconset>
   </property>
   <property name="text">
    <string>Mallado</string>
   </property>
  </action>
  <action name="actionVertices">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/puntos.png</normaloff>:/Imagenes/Iconos/iconos/puntos.png</iconset>
   </property>
   <property name="text">
    <string>Vertices</string>
   </property>
  </action>
  <action name="actionRelleno">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/relleno.png</normaloff>:/Imagenes/Iconos/iconos/relleno.png</iconset>
   </property>
   <property name="text">
    <string>Relleno</string>
   </property>
  </action>
  <action name="actionRelleno_2">
   <property name="text">
    <string>Relleno</string>
   </property>
  </action>
  <action name="actionMallado_2">
   <property name="text">
    <string>Mallado</string>
   </property>
  </action>
  <action name="actionFog">
   <property name="text">
    <string>Fog</string>
   </property>
  </action>
  <action name="actionActivar_Fog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/fog.png</normaloff>:/Imagenes/Iconos/iconos/fog.png</iconset>
   </property>
   <property name="text">
    <string>Fog</string>
   </property>
  </action>
  <action name="actionVertices_2">
   <property name="text">
    <string>Vertices</string>
   </property>
  </action>
  <action name="actionZ_Buffer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/zbuffer.png</normaloff>:/Imagenes/Iconos/iconos/zbuffer.png</iconset>
   </property>
   <property name="text">
    <string>Z-Buffer</string>
   </property>
  </action>
  <action name="actionRotar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/rotar.png</normaloff>:/Imagenes/Iconos/iconos/rotar.png</iconset>
   </property>
   <property name="text">
    <string>Rotar</string>
   </property>
  </action>
  <action name="actionTransladar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/trans.png</normaloff>:/Imagenes/Iconos/iconos/trans.png</iconset>
   </property>
   <property name="text">
    <string>Transladar</string>
   </property>
  </action>
  <action name="actionBoundingBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/bounding.png</normaloff>:/Imagenes/Iconos/iconos/bounding.png</iconset>
   </property>
   <property name="text">
    <string>Bounding Box</string>
   </property>
  </action>
  <action name="actionBounding_Box">
   <property name="text">
    <string>Bounding Box</string>
   </property>
  </action>
  <action name="actionPlano_de_Corte">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/clipping.png</normaloff>:/Imagenes/Iconos/iconos/clipping.png</iconset>
   </property>
   <property name="text">
    <string>Plano de Corte</string>
   </property>
  </action>
  <action name="actionMover_Trasladar_Plano_de_Corte">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/modifClipping.png</normaloff>:/Imagenes/Iconos/iconos/modifClipping.png</iconset>
   </property>
   <property name="text">
    <string>Rotar/Trasladar Plano de Corte</string>
   </property>
  </action>
  <action name="actionNormales">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/normal.png</normaloff>:/Imagenes/Iconos/iconos/normal.png</iconset>
   </property>
   <property name="text">
    <string>Normales</string>
   </property>
  </action>
  <action name="actionNormales_2">
   <property name="text">
    <string>Normales</string>
   </property>
  </action>
  <action name="actionBackface_Culling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/Imagenes/Iconos/iconos/back.png</normaloff>:/Imagenes/Iconos/iconos/back.png</iconset>
   </property>
   <property name="text">
    <string>Backface Culling</string>
   </property>
  </action>
  <action name="actionInformacion_del_Objeto">
   <property name="text">
    <string>Informacion del Objeto</string>
   </property>
  </action>
  <action name="actionOrtogonal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Paralela</string>
   </property>
  </action>
  <action name="actionPerspectiva">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Perspectiva</string>
   </property>
  </action>
  <action name="actionFlikering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flikering</string>
   </property>
  </action>
  <action name="actionNormal_counter_clockwise">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal counter clockwise</string>
   </property>
  </action>
  <action name="actionNormal_clockwise">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal clockwise</string>
   </property>
  </action>
  <action name="actionCargar_Textura">
   <property name="text">
    <string>Cargar Textura</string>
   </property>
  </action>
  <action name="actionActivar_Textura">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Activar Textura</string>
   </property>
  </action>
  <action name="actionEsferico">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Planar</string>
   </property>
  </action>
  <action name="actionCilindrico">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cilindrico</string>
   </property>
  </action>
  <action name="actionLinear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="actionNeares">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nearest</string>
   </property>
  </action>
  <action name="actionClamp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clamp</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
